web/react/utils/utils.jsx

```diff
@@ -753,8 +753,23 @@ module.exports.isValidUsername = function(name) {
     return error;
 };
 
+function checkRTL(string) {
```

Comment:
I'm open to using a library, but it seems like a simple enough and stable enough thing to just use a regex of what is what type.